README.md

```diff
@@ -47,7 +47,8 @@ integration with Rust tooling. This could also be applied to other languages too
 
 Clone the project and run `cargo build --release`.
 
-**NOTE:** Iota needs to be built using the nightly toolchain for now, not stable.
+**NOTE:** Iota needs to be built using the nightly toolchain for now, not stable.<br>
+Run the following commands - `$ rustup install nightly` following which run - `$ rustup default nightly `.<br>
```

Comment:
Could you please use `rustup override set nightly` instead of `rustup default nightly` so that nighlty doesn't get the global default.